docs/tutorials/Bucket/bucket_tutorial.jl

```diff
@@ -180,7 +182,10 @@ z_0b = FT(1e-3);
 # Thermal parameters of soil
 κ_soil = FT(0.7);
 ρc_soil = FT(2e6);
-Δt = FT(3600.0);
+# Simulation start time, end time, and timestep
+t0 = FT(0.0);
+tf = FT(7 * 86400);
+Δt = FT(360.0);
```

Comment:
is this another case where we need a smaller timestep because we arent using adaptive stepping? or could we keep the timestep the same?

Reply:
This was using adaptive timestepping before since that's the default for `ODE.solve`, so I do think we need a smaller timestep.

Reply:
Actually, I just tried running this with the same timestep as before (3600s) and it runs locally, so I'll change it back to that.